repos/omeka-i18n-module/src/Loader/MadocMessageLoader.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+namespace i18n\Loader;
+
+use Digirati\OmekaShared\Utility\OmekaValue;
+use FilesystemIterator;
+use RegexIterator;
+use Symfony\Component\Translation\Loader\PoFileLoader;
+use Zend\I18n\Translator\Loader\RemoteLoaderInterface;
+use Zend\I18n\Translator\TextDomain;
+
+class MadocMessageLoader implements RemoteLoaderInterface
```

Comment:
I think this can be removed, see `LocalizationListener` for new implementation based on the file system